Git Formats/syntax_test_git_config

```diff
@@ -377,14 +381,26 @@ stray-bracket]
 #                           ^  constant.numeric.integer.pretty-formats.git
 #                            ^ punctuation.section.parens.end.pretty-formats.git
 #                             ^^ constant.other.placeholder.pretty-formats.git
+    foo = log --pretty=format:%h
+#                      ^^^^^^ keyword.declaration.pretty-formats.git
+#                            ^ keyword.operator.assignment.option.pretty-formats.git
+#                             ^^ constant.other.placeholder.pretty-formats.git
+    foo = log --pretty=tformat:%h
+#                      ^^^^^^^ keyword.declaration.pretty-formats.git
+#                             ^ keyword.operator.assignment.option.pretty-formats.git
+#                              ^^ constant.other.placeholder.pretty-formats.git
     foo = log --pretty=%a
 #                      ^^ - constant
-    foo = log --pretty=%aD
+    foo = log --pretty=%aDx%cD
 #                      ^^^ constant.other.placeholder.pretty-formats.git
```

Comment:
@michaelblyons I assume you should change this as well, when changing the previous line.